buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/Dependencies.kt

```diff
@@ -119,9 +118,9 @@ fun Project.ideProbe() {
 }
 
 fun Project.jcabiAspects() {
```

Comment:
can you remind me, why we need it?

Reply:
Meh, `@Loggable` to include execution time in logs for methods... albeit after the 3+ years these entire logs seem mostly useless in general anyway :///

Reply:
We don't use them (at least I prefer more case-by-case printlns than the massive spam in logs), and users never attach them to the issues (and I don't remember a single case where having such logs would actually help)

Reply:
that's true 😕